# --- SDE-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
#
# Filename: package/.../fcgi/gcc44-include-cstdio.patch
# Copyright (C) 2009 The OpenSDE Project
#
# More information can be found in the files COPYING and README.
#
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- SDE-COPYRIGHT-NOTE-END ---

--- fcgi-2.4.0/include/fcgio.h.orig	2009-12-11 10:10:47.000000000 +0100
+++ fcgi-2.4.0/include/fcgio.h	2009-12-11 10:18:04.000000000 +0100
@@ -31,6 +31,7 @@
 #define FCGIO_H
 
 #include <iostream>
+#include <cstdio>
 
 #include "fcgiapp.h"
 
